src/object/srv_ec_aggregate.c

```diff
@@ -1379,26 +1427,21 @@ agg_peer_update_ult(void *arg)
 			rc = crt_bulk_create(dss_get_module_info()->dmi_ctx,
 					     &sgl, CRT_BULK_RW, &bulk_hdl);
 			if (rc) {
-				D_ERROR(DF_UOID" pidx %d to peer %d, "
-					"crt_bulk_create "DF_RC"\n",
-					DP_UOID(entry->ae_oid), pidx, peer,
-					DP_RC(rc));
-				goto out_rpc;
+				D_ERROR(DF_UOID" pidx %d to peer %d, crt_bulk_create "DF_RC"\n",
+					DP_UOID(entry->ae_oid), pidx, peer, DP_RC(rc));
+				goto next;
```

Comment:
same, if no any peer updated yet (peer_updated == 0), for this kind of failure maybe better to fail the peer update rather than ignore it?

Reply:
agreed